docs/python_client.md

```diff
@@ -4,72 +4,42 @@ excerpt: Developing using the MemGPT Python client
 category: 6580dab16cade8003f996d17
 ---
 
-The fastest way to integrate MemGPT with your own Python projects is through the `MemGPT` [client class](https://github.com/cpacker/MemGPT/blob/main/memgpt/client/client.py):
+The fastest way to integrate MemGPT with your own Python projects is through the [client class](https://github.com/cpacker/MemGPT/blob/main/memgpt/client/client.py):
 
 ```python
-from memgpt import MemGPT
```

Comment:
FYI this naming scheme was intentional to mimic the OpenAI Python SDK:
https://github.com/openai/openai-python?tab=readme-ov-file#usage

However I don't think it's necessary / I think `create_client` is fine too. LMK what you think.